cli/chaosli/cmd/context.go

```diff
@@ -0,0 +1,393 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/DataDog/chaos-controller/api/v1beta1"
+	"github.com/briandowns/spinner"
+	goyaml "github.com/ghodss/yaml"
+	"github.com/spf13/cobra"
+	"io/ioutil"
+	v1 "k8s.io/api/core/v1"
+	"log"
+	"math"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const MAXTARGETSHOW = 10
+const POD = "pod"
+
+func contextTargetsSize(disruption v1beta1.Disruption) ([]string, error) {
+	spec := disruption.Spec
+	labels := spec.Selector.String()
+	level := spec.Level
+	// If node level, this \n will give us empty namespaces for node which is intended because nodes
+	// do not require namespaces to be described
+	podNamespaces := disruption.ObjectMeta.Namespace
+	rowNames := 1
+
+	if level == "node" {
+		rowNames = 1
+	}
+
+	fmt.Println("Let's look at your targets...")
+
+	s := spinner.New(spinner.CharSets[38], 100*time.Millisecond)
+	s.Start()
+	cmd := fmt.Sprintf("kubectl get %s -n %s -l %s | wc -l", level, podNamespaces, labels)
+	sizeString, err := exec.Command("bash", "-c", cmd).Output()
+
+	if err != nil {
+		return nil, fmt.Errorf("Could not count the number of pods correlating to target selector: %v", err)
+	}
+	size, err := strconv.Atoi(strings.Trim(string(sizeString), "\n"))
+
+	if err != nil {
+		return nil, fmt.Errorf("Could not convert string to integer in context target size: %v", err)
```

Comment:
My ide's linter is upset in a few places in this file because golang errors should not be capitalized at the beginning